src/components/utility/Typography/Typography.module.css

```diff
@@ -17,13 +17,13 @@ h3 {
 
 .h3,
 .body1 {
-  font-size: 0.9rem;
+  font-size: 1.125rem;
```

Comment:
these are supposed to be 18px (at root font size 16).

Reply:
Please revert this. The default styles here are declared for the _mobile_ display size. The desktop font size is in the media query on line 55 (and is using two decimal places after discussion between Poulami and me on the Typography PRs).

Reply:
Sorry about that!